### DISCUSSION TITLE
LXQTBuildTools stripped version

### DISCUSSION THREAD

**Discussion by redtide** [qtilities/sddm-conf#2] [General]
In picom-conf and this project I use a stripped down version of build tools because I need only 3 CMake modules for languages and desktop files translations, avoiding an external dependency (mostly to avoid some version and timing issues with CI I faced previously).

I would like to know if I should rename it to not make confusion with the original and make it a small git submodule for projects that uses it. Basically the files are same as original, except in `LXQtTranslateTs.cmake`, I just modified the hardcoded Qt version variable. Or maybe there is a better solution?

**Comment by tsujan** [comment]
I think renaming is the correct way to go.

BTW, I was very busy with different projects in recent days; sadly, I couldn't find time to try sddm-conf, although it was compiled well. I'll try it as soon as I find some time and will tell you about the result here.

**Reply by tsujan** [reply]
Oh, sorry, I misunderstood your comment! You meant the folder `cmake/LXQTBuildTools` inside the source, while I thought that you made a separate repository with the name `lxqt-build-tools`.

As long as it's there and not a separate source, I don't think it could create any confusion.

**Reply by redtide** [reply]
Nope, you were correct, in order to not duplicate files for each project I wanted to add a common repository to use as tiny git submodule, in case files will need an update it can be done in one place. I guess I could call it "QtTranslateModules" and add a README.md that explains that they are derived from LXQTBuildTools, not the original ones.

No hurry about sddm-conf.

**Comment by redtide** [comment]
Closing, the next versions will use the only needed parts of upstream tools and will be a build time only dependency (for the distros like Arch that supports it).
